### DISCUSSION TITLE
dive behaviour when manuel mode on

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav-configurator#2217] [General]
hello, who can tell me how finda solution on a fixed wing: autotrim and autotune has been done: ACRO mode is good, but when i toggle in manuel mode, wing is diving with 30 to 45 ° angle.
inav 7.1.2
matek f405 TE
why?
best regards

**Comment by mmosca** [comment]
In manual, INAV is only doing the mixing for you. The rest is 100% your inputs and your aircraft mechanical setup.

Check your auto trim results, cg, surface alignment, etc....

**Comment by sensei-hacker** [comment]
Does your phone have an elevator (on a tail) or does it have elevons?

**Reply by samsab1** [reply]
hello it's a S800 Serpent wing, so elevon

**Comment by sensei-hacker** [comment]
So elevons, then. How much reflex do you have with the sticks centered?

**Comment by MartinHugh** [comment]
Note that the aircraft needs to fly well in manual for the FC to perform at its best.  *Not* the other way around. 
You should check the CG is correct and adjust mechanical trims in order to get this flying nicely in manual, with the servos close to 1500.

**Comment by samsab1** [comment]
what surprises me is that this s800 flying wing flew very well in manual, although I have problems in acro mode for other reasons. so I have changed the parameters related to the PID many times without solving the problem, (see https://www.facebook.com/share/v/BRrToR2yS51L1Eoy/) and since then, I have observed a systematic sudden dive when switching to manual, without, on the ground, there being the slightest change in steering between Acro and manual mode.
in standard ACRO mode (at medium speed before my wooble problem appears in ACRO mode at higher speed), this wing flies perfectly with elevons in the position expected for a stabilized flight. this position does not vary (on the ground) when switching from ACRO to MANUAL, and vice versa; so why this dive?, the center of gravity is good, the controls are ok, the servos are ok.

**Reply by sensei-hacker** [reply]
There are three possible causes. These correspond to the three forces acting on the aircraft in flight - aerodynamic, gravity, and thrust.

How much reflex does it have with the sticks centered, please?

**Comment by samsab1** [comment]
sensei, thanks to take care of me. finally i have restored a previous backup of this wing, and ii solved both pb (dive in manual and wooble in ACRO, without knowing where was the both pb....so strange sometimes
